app/controllers/offers_controller.rb

```diff
@@ -130,6 +133,19 @@ def update_print_status(offers)
     end
   end
 
+  def set_status_helper(params)
```

Comment:
when user presses "accept" this updates the status of the offer

set_status_following_contract_accept_or_reject_or_withdraw ???